packages/html2sb-compiler/test/fixtures/invalid-list-in-list.html

```diff
@@ -0,0 +1 @@
+<ul><ul></ul></ul>
```

Comment:
`<ul>` can't be in `<ul>` as HTML syntax rule..., but this test case is added as [Forgiving Parsing](https://en.wikipedia.org/wiki/Tag_soup#:~:text=XHTML%20article.-,HTML5).

By this pull request, web pages such as https://faq.airpayment.jp/hc/ja/articles/360013687633 will be converted as much as possible.
[![Image from Gyazo](https://t.gyazo.com/teams/nota/a58f091da632ec690cf79b094eab2fad.png)](https://nota.gyazo.com/a58f091da632ec690cf79b094eab2fad)

Reply:
good  as an edge case 😄